task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +291,12 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let arrNum = num.toString().split('');
+    let sum = 0;
+    for (let i = 0; i < arrNum.length; i++) {
+        sum += +arrNum[i];
+    }
+    return sum.toString().length === 2 ? +sum.toString().charAt(0) + +sum.toString().charAt(1) : sum;
```

Comment:
In future try to avoid this code style, if-else is more readable than multiple conditional operators ? :